modules/microshift-auto-apply-manifests.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-auto-apply-manifests_{context}"]
+= Auto-applying manifests
+
+When you run `kustomize`, {product-title} searches `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k command` to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-name} workloads.
```

Comment:
What kind of thing is `kustomize`? I guessed "command" but idk :)

Per [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references):

>Use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This helps with clarity and readability.

```suggestion
When you run the `kustomize` command, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/manifests` directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the manifests to the cluster. Providing multiple directories allows a flexible method of managing {product-title} workloads.
```